### PR TITLE
Calculate ideal weight

### PR DESCRIPTION
**Pull request by TPANCHULIDZE**

### CODE REVIEW

File: ideal_weight.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+IDEAL_WEIGHT = 110
+
+class Human
+  def initialize(name, height)
+    @name = name
+    @height = height
+  end
+
+  def ideal_weight(name, height)
```
Comment (alexshvedko) [inline ideal_weight.rb]: you should use variables defined in the constructor. I think, it does not work

Comment (TPANCHULIDZE) [reply ideal_weight.rb]: I deleted this variables in sublime text i did't  know why this is seems here, i will check my code tomorrow

File: ideal_weight.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# calculate ideal weight
+
+IDEAL_WEIGHT = 110
+
+class Human
```
Comment (alexshvedko) [inline ideal_weight.rb:5]: Screenshot 2022-04-01 at 11 10 53

Comment (alexshvedko) [reply ideal_weight.rb:5]: add validations

**Review by alexshvedko — APPROVED** [review]

**Pull request merged**